desktop/src/ui/preload.ts

```diff
@@ -18,6 +17,25 @@ export function concatLink (host: string, path: string): string {
   }
 }
 
+async function loadServerConfig (url: string): Promise<any> {
```

Comment:
May be we could check for error?

Reply:
You mean only retry for this specific error or somehow handle when all retries fail?